_doc/developer/general/documentation-style-guide.md

```diff
@@ -0,0 +1,355 @@
+
+The documentation is a volunteer community effort. People like you are constantly working to make it better. If you notice something that can be fixed or improved, please [edit the documentation](/doc/how-to-edit-the-documentation/)!
+
+This page explains the standards we follow for writing, formatting, and organizing the documentation. Please follow these guidelines and conventions when editing the documentation. For the standards governing the website as a whole, please see the [website style guide](/doc/website-style-guide).
+
+## Markdown conventions
+
+All the documentation is written in Markdown for maximum accessibility. When making contributions, please observe the following style conventions. If you're not familiar with Markdown syntax, [this](https://daringfireball.net/projects/markdown/) is a great resource.
+
+### Hyperlink syntax
+
+Use non-reference-style links like `[website](https://example.com/)`. Do *not* use reference-style links like `[website][example]`, `[website][]` or `[website]`. This facilitates the localization process.
+
+### Relative vs. absolute links
+
+Always use relative rather than absolute paths for internal website links. For example, use `/doc/documentation-style-guide/` instead of `https://www.qubes-os.org/doc/documentation-style-guide/`.
+
+You may use absolute URLs in the following cases:
+
+- External links
+- URLs that appear inside code blocks (e.g., in comments and document templates, and the plain text reproductions of [QSBs](/security/qsb/) and [Canaries](/security/canary/)), since they're not hyperlinks
+- Git repo files like `README.md` and `CONTRIBUTING.md`, since they're not part of the website itself but rather of the auxiliary infrastructure supporting the website
+
+This rule is important because using absolute URLs for internal website links breaks:
+
+- Serving the website offline
+- Website localization
+- Generating offline documentation
+- Automatically redirecting Tor Browser visitors to the correct page on the onion service mirror
+
+### Image linking
+
+See [how to add images](https://doc.qubes-os.org/en/latest/developer/general/how-to-edit-the-documentation.html#how-to-add-images) for the required syntax. This will make the image a hyperlink to the image file, allowing the reader to click on the image in order to view the full image by itself. This is important. Following best practices, our website has a responsive design, which allows the website to render appropriately across all screen sizes. When viewing this page on a smaller screen, such as on a mobile device, the image will automatically shrink down to fit the screen. If visitors cannot click on the image to view it in full size, then, depending on their device, they may have no way see the details in the image clearly.
```

Comment:
This link redirects to
https://www.qubes-os.org/doc/how-to-edit-the-documentation/
missing the anchor/fragment `#how-to-add-images`.

In order to fix it, I suggest to link to
https://www.qubes-os.org/doc/how-to-edit-the-documentation/#how-to-add-images
directly or using magic:
`/doc/how-to-edit-the-documentation/#how-to-add-images`

However, I did not test my suggestion.